spec/controllers/proposals_controller_spec.rb

```diff
@@ -194,6 +194,7 @@
       es_execute_with_retries 3 do
         get :download, text: "Work Order", format: "csv"
         expect(response.body).to include "Work Order 29"
+        expect(response.body).to include proposals.last.client_data.approving_official.display_name
```

Comment:
oh! cool. didn't realize that. was thinking about the `full_name` method. I feel like we should use `display_name` everywhere although I realize not everyone agrees with that